core/store/migrations/migration1560433987/migrate.go

```diff
@@ -0,0 +1,20 @@
+package migration1560433987
+
+import (
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+)
+
+// Migrate changes the confirmations column to be nullable and default to null.
+// Must be done in a roundabout way because SQLite does not support altering a column's
+// default or nullability.
+func Migrate(tx *gorm.DB) error {
+	err := tx.Exec(`
+		ALTER TABLE task_runs ADD COLUMN confirmations_new1560433987 bigint;
+		UPDATE task_runs SET confirmations_new1560433987 = confirmations;
+		ALTER TABLE task_runs RENAME COLUMN confirmations TO confirmations_old1560433987;
+		ALTER TABLE task_runs RENAME COLUMN confirmations_new1560433987 TO confirmations;
```

Comment:
I have yet to create a whole new table to remove the `confirmations_new1560433987`, as required by sqlite.